Libraries/Image/AssetSourceResolver.js

```diff
@@ -22,7 +22,7 @@ import type {PackagerAsset} from 'AssetRegistry';
 const PixelRatio = require('PixelRatio');
 const Platform = require('Platform');
 
-const assetPathUtils = require('../../local-cli/bundle/assetPathUtils');
```

Comment:
assetPathUtils are not limited to images. They also impact all other types of assets.

Reply:
Right, but my understanding is that the "impact" is more present in the `local-cli` context where the utils are used to generate a path to any type of asset.

On the React Native side of things, this file is only used in `Image` module. Anyway, this shouldn't be an issue now that the `local-cli` folder will not be removed. I am going to revert this change and w can leave this to be refactored at some point.